apps/researcher/src/app/[locale]/communities/community-card.tsx

```diff
@@ -12,7 +13,15 @@ interface MembershipCountProps {
 
 async function MembershipCount({communityId, locale}: MembershipCountProps) {
   const t = await getTranslator(locale, 'Communities');
-  const memberships = await getMemberships(communityId);
+
+  let memberships = [];
+  try {
+    memberships = await getMemberships(communityId);
+  } catch (error) {
```

Comment:
Use `err` instead, to be in line with line 24 in file `apps/researcher/src/app/[locale]/communities/[slug]/page.tsx`?